ansible/roles/machine/provision_multidomain/templates/ipa-test-multidomain-config.yaml

```diff
@@ -0,0 +1,51 @@
+ad_admin_name: Administrator
```

Comment:
Instead of defining a new template, you can enhance the existing one ipa-test-config.yaml and add the following lines just before the lines for AD:
```
{% if trusted_domain %}
- hosts:
{% for host in groups['all'] %}
{% if host.startswith('trustedmaster') or host.startswith('trustedreplica') or host.startswith('trustedclient') %}
    - external_hostname: {{ hostvars[host]['inventory_hostname'] }}.trustedipa.test
      name: {{ hostvars[host]['inventory_hostname'] }}.trustedipa.test
      ip: {{ hostvars[host]['ansible_default_ipv4']['address'] }}
{% if host.startswith('trustedmaster') %}
      role: master
{% endif %}
{% if host.startswith('trustedreplica') %}
      role: replica
{% endif %}
{% if host.startswith('trustedclient') %}
      role: client
{% endif %}
{% endif %}
{% endfor %}
  name: trustedipa.test
  type: TRUSTED_IPA
{% endif %}
```